external-table/src/pxfutils.c

```diff
@@ -116,6 +128,85 @@ get_pxf_port(void)
 	return port;
 }
 
+const char *
+get_pxf_ssl_keypasswd(void)
+{
+	const char *proto = getenv(ENV_PXF_SSL_KEYPASSWD);
+
+	if (proto == NULL) 
+	{
+		proto = PXF_DEFAULT_SSL_KEYPASSWD;
+	}
+
+	return proto;
+}
+
+const char *
+get_pxf_ssl_cacert(void)
+{
+	const char *cacert = getenv(ENV_PXF_SSL_CACERT);
+
+	if (cacert == NULL) 
+	{
+		cacert = PXF_DEFAULT_SSL_CACERT;
+	}
+
+	return cacert;
+}
+
+const char *
+get_pxf_ssl_cert(void)
+{
+	const char *cert = getenv(ENV_PXF_SSL_CERT);
+
+	if (cert == NULL)
+	{
+		cert = PXF_DEFAULT_SSL_CERT;
+	}
+
+	return cert;
+}
+
+const char *
+get_pxf_ssl_key(void)
+{
+	const char *key = getenv(ENV_PXF_SSL_KEY);
+
+	if (key == NULL)
+	{
+		key = PXF_DEFAULT_SSL_KEY;
+	}
+
+	return key;
+}
+
+const char *
+get_pxf_ssl_certtype(void)
+{
+	const char *certtype = getenv(ENV_PXF_SSL_CERT_TYPE);
+
+	if (certtype == NULL)
+	{
+		certtype = PXF_DEFAULT_SSL_CERT_TYPE;
+	}
+
+	return certtype;
+}
+
+long
+get_pxf_ssl_verifypeer(void)
+{
+	const char *verify_peer_var = getenv(ENV_PXF_SSL_VERIFY_PEER);
+	long verifypeer = PXF_DEFAULT_SSL_VERIFY_PEER; 
+
+	if (verify_peer_var != NULL) 
+	{
+		verifypeer = atol(verify_peer_var);
+	}
+
+	return verifypeer;
```

Comment:
I suggest to remove the verifypeer variable
```suggestion

	if (verify_peer_var != NULL) 
	{
		return atol(verify_peer_var);
	}

	return PXF_DEFAULT_SSL_VERIFY_PEER;
```

Reply:
This code rewritten now.